src/actions/arcActions.ts

```diff
@@ -30,6 +30,16 @@ import { Dispatch } from "redux";
 import { ExecutionState, TransactionService } from "@daostack/arc.js";
 import { showOperation } from "./operationsActions";
 import { OperationsStatus } from "reducers/operations";
+import * as moment from "moment";
+
+function proposalEnded(proposal: IProposalState) {
```

Comment:
@dkent600 this seems like great functionality  to move into arc.js at some point